openmdao_book/other_useful_docs/api_translation.ipynb

```diff
@@ -0,0 +1,2191 @@
+{
```

Comment:
<p>This looks different than what is in the existing docs.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO_Book/pull/78/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>Yeah I added this so the code would run but fair point, reverted back to previous code </p>